### DISCUSSION TITLE
how to fine-tuning own dataset with siamrpn_r50_l234_dwxcorr_lt

### DISCUSSION THREAD

**Discussion by yustaub** [STVIR/pysot#343] [General]
nice work!! But I want to train my own dataset with siamrpn_r50_l234_dwxcorr_lt, I have download the model ,and I want to fine-tune my own dataset with it , where should I change? very appreciate for your reply!

**Comment by HanlynnKe** [comment]
Hi, I've done training my own TIR dataset with pysot framework. You just need to add your dataset at /pysot/core/config.py and also the yaml in the directory where you start training. I make my dataset in from of VID2015 btw

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing this issue for now. 
Please create a new issue should you have other questions.

**Comment by haishibei** [comment]
> Hi, I've done training my own TIR dataset with pysot framework. You just need to add your dataset at /pysot/core/config.py and also the yaml in the directory where you start training. I make my dataset in from of VID2015 btw

I wonder how about the result on your own dataset? Can you share it with us?

**Comment by HanlynnKe** [comment]
> > Hi, I've done training my own TIR dataset with pysot framework. You just need to add your dataset at /pysot/core/config.py and also the yaml in the directory where you start training. I make my dataset in from of VID2015 btw
> 
> I wonder how about the result on your own dataset? Can you share it with us?

I trained SiamRPN++ (ResNet-50 as backbone) in my thermal infrared dataset and tested on PTB-TIR dataset. The results are as follows:
Accuracy | Robustness | Loss | EAO
---|---|---|---
0.428|0.350|92|0.264

**Comment by haishibei** [comment]
> > > Hi, I've done training my own TIR dataset with pysot framework. You just need to add your dataset at /pysot/core/config.py and also the yaml in the directory where you start training. I make my dataset in from of VID2015 btw
> > 
> > 
> > I wonder how about the result on your own dataset? Can you share it with us?
> 
> I trained SiamRPN++ (ResNet-50 as backbone) in my thermal infrared dataset and tested on PTB-TIR dataset. The results are as follows:
> Accuracy 	Robustness 	Loss 	EAO
> 0.428 	0.350 	92 	0.264
Thanks for your generous sharing
